Automattic/wp-calypso
# Stats: Add Tracks to MobilePromoCard component

#### Proposed Changes

Adds support for Tracks to the various links. We do not track the QR code as there isn't any browser event for someone scanning a QR code. The tracked events are:

**FOR JETPACK:**
* `calypso_stats_mobile_cta_jetpack_view` — The Jetpack CTA is rendered.
* `calypso_stats_mobile_cta_jetpack_click` — User clicks link to jetpack.com/apps
* `calypso_stats_mobile_cta_jetpack_apple_click` — User clicks App Store badge for Jetpack app
* `calypso_stats_mobile_cta_jetpack_google_click` — User clicks Google Play badge for Jetpack app

**FOR WOO**
* `calypso_stats_mobile_cta_woo_view` — The Woo CTA is rendered.
* `calypso_stats_mobile_cta_woo_click` — User clicks link to woo.com/mobile
* `calypso_stats_mobile_cta_woo_apple_click` — User clicks App Store badge for Woo app
* `calypso_stats_mobile_cta_woo_google_click` — User clicks Google Play badge for Woo app

Additionally, I've removed the raw URLs in favour of the jetpack.com redirects. This allows us to update the URLs server-side without having to make code changes. Applies to QR codes as well.

#### Testing Instructions



* Check out this branch.
* Run `yarn workspace @automattic/components run storybook`.
* Visit the Mobile Promo Card entry.
* Confirm the icons, text, and QR codes display correctly.
* Confirm the QR codes link out correctly.
* Use the Chrome dev tools and change to the "iPhone 12 Pro" setting. Refresh the view and confirm you get an App Store badge that links to the app page.
* Change to the "Pixel 5" setting and refresh again. Confirm you get a Play Store badge and that it's wired up correctly.
* Wait 24 hours to confirm our events are being recorded correctly?!

#### Pre-merge Checklist





Related to #70463.
